matfyz_forum/matfyz_forum/asgi.py

```diff
@@ -0,0 +1,16 @@
+"""
```

Comment:
I appreciate the usage of newer standards here. However make sure, that the server hosting the application supports such a service.

Reply:
This file was auto-generated by django I didn't create on purpose since I don't think we will use the async features of django. If you want to use it let me know because further configuration( Middleware selection ) will be needed. Also bear in mind that ORM does not support async and it needs to be translated. Thus if we agree on not supporting async, I will delete it.

Reply:
Yes, we will be using asynchronous operations, yet I do not know entirly what options Django offers, however we will be using Cellery and Redis, therefore the dependency on the django is not strictly required, yet desired. For now, just keep it there.